charts/datadog-operator/values.yaml

```diff
@@ -15,6 +15,14 @@ apiKeyExistingSecret:  # <DATADOG_API_KEY_SECRET>
 # appKey -- Your Datadog APP key
 appKey:  # <DATADOG_APP_KEY>
 
+# site -- The site of the Datadog intake to send data to
```

Comment:
could you add a link to the [documentation](https://docs.datadoghq.com/getting_started/site/) on this line 
```suggestion
# site -- The site of the Datadog intake to send data to (documentation: https://docs.datadoghq.com/getting_started/site/)
```